openshift/e2e-common.sh

```diff
@@ -192,9 +167,9 @@ function run_e2e_tests(){
   fi
 
   header "Running tests with Multi Tenant Channel Based Broker"
-  oc apply -f config/core/configmaps/default-broker.yaml || return 1
+  oc apply -f config/core/configmaps/default-broker.yaml || return 3
```

Comment:
perhaps we patch ? instead of throwing files around ? 

e..g as here: https://github.com/openshift/knative-eventing/pull/659  ?

Reply:
OK. Done.